src/assets/toolkit/styles/components/message.css

```diff
@@ -0,0 +1,38 @@
+/** @define Message */
+
+:root {
+  --Message-padding: var(--space-xs) 2.5em;
+  --Message-background: var(--color-gray);
+  --Message-offline-background: var(--color-orange);
+  --Message-offline-color: var(--color-white);
```

Comment:
It is just barely failing. We also use this color combo on buttons, so if it is a big concern, we'll have to reconsider a few things, maybe even making `--color-orange` darker.